fix.py

```
@@ -0,0 +1,23 @@
+content = open('game/models.py', 'w')
```

Ruff (0.15.12)

[warning] Use a context manager for opening files

(SIM115)